config/initializers/honeycomb.rb

```diff
@@ -0,0 +1,14 @@
+Honeycomb.configure do |config|
+  config.write_key = ENV.fetch('HONEYCOMB_WRITEKEY', 'd5529d94d4e989bb1fc1b1cfc559ebba')
```

Comment:
Is this a secret we should be protecting?

Reply:
ah yup good catch